seeds/data/profiles.json

```diff
@@ -29,6 +29,21 @@
       }
     ]
   },
+  {
+    "id": "0b2d1c52-f8e4-4d16-b58c-0ce80ca58d0c",
+    "userId": "ba836eae-28b7-408e-99e5-dae65d40db4d",
+    "title": "Dr",
+    "firstName": "Neil",
+    "lastName": "Down",
+    "dob": "1962-07-19",
+    "email": "ntco@example.com",
+    "roles": [
+      { "establishmentId": 8201, "type": "ntco" }
+    ],
+    "permissions": [
+      { "establishmentId": 8201, "role": "admin" }
```

Comment:
I might suggest making them _not_ an admin so that it's clear that the approval task is down to NTCO-ness and not admin-ness.